extensions/vscode/package.json.in

```diff
@@ -33,38 +34,36 @@
 	"main": "./lsp-client/out/clientNode",
 	"browser": "./lsp-client/out/clientWeb",
 	"contributes": {
-		"configuration": [
-			{
-				"id": "wakeLanguageServer",
-				"title": "Wake",
-				"type": "object",
-				"properties": {
-					"wakeLanguageServer.pathToWakeStandardLibrary": {
-						"default": "",
-						"description": "If there is a valid share/wake/lib/ at the root of your project, that library is used to provide diagnostics. If not, the path that you provide here is used.\nWith no path provided, the library that comes with the extension is used.\n\nThis setting has no effect when running in a browser.",
-						"scope": "window",
-						"type": "string"
-					},
-					"wakeLanguageServer.trace.server": {
-						"default": "off",
-						"description": "Traces the communication between VS Code and the language server.",
-						"enum": [
-							"off",
-							"messages",
-							"verbose"
-						],
-						"scope": "window",
-						"type": "string"
-					},
-					"wakeTimeline.pathToWakeBinary": {
-						"default": "",
-						"description": "The path to the wake binary that is used to create the timeline. If invalid, the timeline functionality is not provided.",
-						"scope": "window",
-						"type": "string"
-					}
-				}
-			}
-		],
+		"configuration": {
+            "id": "wakeLanguageServer",
+            "title": "Wake",
```

Comment:
This file seems to mainly use tabs, so much as I like spaces myself, they seem to misalign it.

Reply:
Arrays and lists are legal, the configuration didn't use to be an array though, which is why I changed it. I changed it back now

Reply:
The file is partly spaces and partly tabs. Should I rewrite it to use tabs only?

Reply:
If it's already a mix, then let's not fold it into the extension fix.  If you wanted to standardize, I wouldn't complain, but it's probably not required, and should be its own PR if you wanted to do so.

Reply:
Or if you've got the commit already ready, that works too! 😁

Reply:
I pushed the commit!